jboss/container/wildfly/launch/messaging/added/launch/messaging.sh

```diff
@@ -275,7 +275,7 @@ EOF
 # $4 - default connection factory - java:jboss/DefaultJMSConnectionFactory
 # <!-- ##AMQ_POOLED_CONNECTION_FACTORY## -->
 function generate_remote_artemis_connection_factory() {
-    echo "<pooled-connection-factory user=\"${2}\" password=\"${3}\" name=\"${1}\" entries=\"java:/JmsXA java:/RemoteJmsXA java:jboss/RemoteJmsXA ${4}\" connectors=\"netty-remote-throughput\" transaction=\"xa\"/>" | sed -e ':a;N;$!ba;s|\n|\\n|g'
+    echo "<pooled-connection-factory user=\"${2}\" password=\"${3}\" name=\"${1}\" entries=\"${4} java:/JmsXA java:/RemoteJmsXA java:jboss/RemoteJmsXA\" connectors=\"netty-remote-throughput\" transaction=\"xa\"/>" | sed -e ':a;N;$!ba;s|\n|\\n|g'
```

Comment:
@ehsavoie why do we need that change? Is it a new feature that the 1st entry is significant and considered as a default?

It seems to me that we should fix either the `ee` subsystem or the `messagin-activemq` subsystem so that the default JMS ConnectionFactory entry is properly resolved but until now the order of the entries has never been significant.